packages/shared/src/lib/auxiliary/wallet-connect/utils/updateActiveSessionsToActiveProfile.ts

```diff
@@ -0,0 +1,22 @@
+import { SUPPORTED_EVENTS, SUPPORTED_METHODS } from '../constants'
+import { getWalletClient } from '../stores'
+import { getAllEvmAddresses } from './getAllEvmAddresses'
+
+export async function updateActiveSessionsToActiveProfile(): Promise<void> {
+    const chains = ['eip155:1', 'eip155:5']
+    const addresses: string[] = getAllEvmAddresses(chains)
```

Comment:
No need to type constants if the type is returned from the function.
```suggestion
    const addresses = getAllEvmAddresses(chains)
```